backend/src/services/v2/smtp/templates/releaseReviewRequest.ts

```diff
@@ -4,17 +4,26 @@ import mjml2html from 'mjml'
 import { ReviewKindKeys } from '../../../../types/v2/enums.js'
 import { BaseEmailTemplate, IEmailTemplate } from './baseEmailTemplate.js'
 
-export class ReleaseReviewRequest extends BaseEmailTemplate implements IEmailTemplate {
-  getSubject(resourceName: string): string {
-    return dedent(`
-    You have been requested to review '${resourceName}' on Bailo
+export class ReleaseReviewRequestEmail extends BaseEmailTemplate implements IEmailTemplate {
```

Comment:
This seems over complicated.  I think I now understand why you're using both `extends` and `implements` at the same time.  

```typescript
interface Email {
  subject: string
  text: string
  html: string
  to: string
}

function emailWrapper(body: string) {
  ...
}

function sendEmail(email: Email) {
  ...
}

function sendReviewRequestNotifications(to: string, model: ModelDoc, ...) {
  const subject = `${reviewerRole.toUpperCase()}: You have been requested to review '${resourceName}' on Bailo`
  const text = dedent(`
    // some text
  `)

  const html = emailWrapper(dedent(`
    // some html
  `))

  await sendEmail({ to, subject, html, text })
}

```

Seems much more understandable, without losing any of our type safety.

Reply:
I may be misunderstanding what you're suggesting here but it seems like you're wanting to include all of the static email content like the HTML inside `sendReviewRequestNotifications` which is in the `smtp` service? I think this is a bad idea and the static content specific to our different emails should not be included in the generic `smtp` service